test/sentence-length-html-test.ts

```diff
@@ -0,0 +1,54 @@
+import TextLintTester from "textlint-tester";
+import rule from "../src/sentence-length";
+
+const tester = new TextLintTester();
+
+(async function testHTML() {
+    const htmlPlugin = (await import("textlint-plugin-html")).default;
```

Comment:
HTML plugin is pure ESM

Reply:
Throw following error when just `import ... from "textlint-plugin-html"` from CJS.

```
$ textlint-scripts test

TypeError: Unknown file extension ".ts" for /Users/azu/ghq/github.com/textlint-rule/textlint-rule-sentence-length/test/sentence-length-test.ts
    at new NodeError (node:internal/errors:405:5)
    at Object.getFileProtocolModuleFormat [as file:] (node:internal/modules/esm/get_format:79:11)
    at defaultGetFormat (node:internal/modules/esm/get_format:124:36)
    at defaultLoad (node:internal/modules/esm/load:89:20)
    at nextLoad (node:internal/modules/esm/loader:163:28)
    at ESMLoader.load (node:internal/modules/esm/loader:603:26)
    at ESMLoader.moduleProvider (node:internal/modules/esm/loader:457:22)
    at new ModuleJob (node:internal/modules/esm/module_job:64:26)
    at ESMLoader.#createModuleJob (node:internal/modules/esm/loader:480:17)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:434:34)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
``